modules/oe_starter_content_event/src/Plugin/Field/FieldFormatter/EventLinkExternalFormatter.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Drupal\oe_starter_content_event\Plugin\Field\FieldFormatter;
+
+use Drupal\Core\Field\FieldItemListInterface;
+use Drupal\link\Plugin\Field\FieldFormatter\LinkFormatter;
+
+/**
+ * Plugin implementation of the 'link_separate' formatter.
+ *
+ * @FieldFormatter(
+ *   id = "event_link_external",
+ *   label = @Translation("Event links external opens in new tab"),
```

Comment:
it may or may not open in next tab, and the link already has a target blank setting, rather mention why this is different, something more representative.